### DISCUSSION TITLE
fsleyes doesn't fully launch

### DISCUSSION THREAD

**Discussion by cathbridges** [neuropoly/axondeepseg#446] [Help needed] [answered]
I've completed all the installation steps on the website for the development version of AxonDeepSeg via git clone. I have also installed fsleyes. I have run both checks without errors. When I run the code `fsleyes` for the GUI in the Terminal of my MacBook Air running macOS Mojave, a python icon pops up but the application never opens.  I'm not sure what's going wrong. Please let me know if you need more information. I'm new to GitHub. 

Are there minimum requirements for FSLeyes to run?

**Comment by jcohenadad** [comment, marked as answer]
Hi @cathbridges ! Thank you for reaching out. I am wondering if you are running into [this issue](https://users.fmrib.ox.ac.uk/~paulmc/fsleyes/userdoc/latest/troubleshooting.html#xquartz-fsleyes-doesn-t-start-and-just-shows-an-error). Could you please copy/paste the content of the terminal (including the command you enter for launching fsleyes).

**Reply by cathbridges** [reply]
Thank you for your reply!

![Screen Shot 2021-01-30 at 7 21 51 PM](https://user-images.githubusercontent.com/78279758/106371262-6d675280-6330-11eb-8c44-273d2b64e09f.png)

Here is the code I use:
```
Last login: Sat Jan 30 16:38:27 on ttys000
(base) Millies-Mac:~ catherinebridges$ conda activate ads_venv
(ads_venv) Millies-Mac:~ catherinebridges$ cd axondeepseg
(ads_venv) Millies-Mac:axondeepseg catherinebridges$ fsleyes
```
I don't get any errors.

**Reply by Stoyan-I-A** [reply]
@cathbridges It is hard to figure out what is wrong with the installation when we don't have an output on the console 🙁. Later today, I will write a small set of instructions that you can use to install FSLeyes on its own on a new environment without AxonDeepSeg. If the problem doesn't happen, we will have to test some methods to find how to install AxonDeepSeg on your machine. If the problem still happens without installing AxonDeepSeg, then I will have to redirect you to [the maintainers of FSLeyes](https://github.com/pauldmccarthy/fsleyes).

I will post the instructions on this discussion later today.

**Reply by Stoyan-I-A** [reply]
@cathbridges Okay, try this:
1. Open a terminal and create a new virtual environment using conda:
`conda create -n installation_test python=3.7`
2. Activate the environment:
`conda activate installation_test`
3. Install FSLeyes using conda-forge:
`yes | conda install -c conda-forge fsleyes`
_Note: We won't be installing the same version of FSLeyes as our documentation suggests. This is so we can report the issue to the maintainers of FSLeyes if the problem still happens with a version that they currently maintain._
4. Try to launch FSLeyes:
`fsleyes`
Does it launch correctly?

**Reply by cathbridges** [reply]
Thank you for these instructions! When I run `fsleyes`, it does the same thing as before. A python window opens but the application does not and there's no errors. Looks like it's an issue with FSLeyes.

When I force quit python, I see this:
(installation_test) Millies-Mac:~ catherinebridges$ fsleyes
/Users/catherinebridges/miniconda/envs/installation_test/bin/pythonw: line 3: 64410 Terminated: 15          /Users/catherinebridges/miniconda/envs/installation_test/python.app/Contents/MacOS/python "$@"

**Reply by jcohenadad** [reply]
@cathbridges Could you please open an issue in the [FSLeyes issue tracker](https://github.com/pauldmccarthy/fsleyes/issues) and cross-reference it here? Many thanks.

**Reply by cathbridges** [reply]
Done.  #pauldmccarthy/fsleyes#66

**Reply by Stoyan-I-A** [reply]
In https://github.com/pauldmccarthy/fsleyes/issues/66, Paul found that this issue can be fixed by downgrading the wxPython version.

@cathbridges, You could now try to follow [our documentation](https://axondeepseg.readthedocs.io/en/latest/documentation.html#installation) to install AxonDeepSeg and just downgrade the wxPython version using the command `conda install -c conda-forge "wxpython=4.0.*"` after installing FSLeyes. Let us know if it works now.

**Reply by cathbridges** [reply]
Everything works now! Thanks!
